test/modify_yaml_tests.py

```diff
@@ -0,0 +1,37 @@
+""" Tests for the modify_yaml Ansible module. """
+# pylint: disable=missing-docstring,invalid-name
+
+import os
+import sys
+import unittest
+
+sys.path = [os.path.abspath(os.path.dirname(__file__) + "/../library/")] + sys.path
+
+# pylint: disable=import-error,wrong-import-position
```

Comment:
Someone, in their infinite wisdom, decided they should constantly change the pylint keys, and make it fail if it doesn't recognize one. Must not be present in whatever env we're running pylint for CI. I'll remove it.